src/Cms/Language.php

```diff
@@ -566,8 +566,13 @@ public function update(array $props = null): static
 	 * Returns a language variable object
 	 * for the key in the translations array
 	 */
-	public function variable(string $key): LanguageVariable
+	public function variable(string $key, bool $decode = false): LanguageVariable
 	{
+		// decode for encoded strings
+		if ($decode === true) {
+			$key = base64_decode(urldecode($key));
```

Comment:
`encodeURIComponent` percent-escapes any special character. Its mirror on the PHP side is `rawurldecode()`. `urldecode()` converts a `+` to a space, `rawurldecode()` doesn't.